src/Stratis.Bitcoin/P2P/Protocol/Payloads/BitcoinSerializablePayload.cs

```diff
@@ -4,11 +4,12 @@ namespace Stratis.Bitcoin.P2P.Protocol.Payloads
 {
     public class BitcoinSerializablePayload<T> : Payload where T : IBitcoinSerializable, new()
     {
-        private T obj = new T();
+        private T obj;
         public T Obj { get { return this.obj; } set { this.obj = value; } }
 
         public BitcoinSerializablePayload()
         {
+            //obj = new T();
```

Comment:
erm?

Reply:
yes focus on logic breaking right now I will delete stuff like this next